### ISSUE TITLE
In High contrast Aquatic mode, Line charts present in the page are not visible in Windows.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#23212]
**Test Environment:** 
OS: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26120.3380)
Browser: Microsoft edge Version 137.0.3296.83
URL: https://www.highcharts.com/demo/highcharts/line-chart

**User Impact:**
Low vision user depending on high contrast will not be able to get the chart information if they are not visible.

**Observation:**
This issue is not observed in Desert mode.

**Pre-Requisite:**
Turn on High Contrast Aquatic (Go to settings->Contrast Themes->High Contrast aquatic).

**Steps to Reproduce:**   

1. Open the URL - https://www.highcharts.com/demo/highcharts/line-chart in latest edge browser.
2. 'Line charts' page will appear.
3. Press tab key to reach to the 'Line chart'.
4. Verify whether In High contrast Aquatic mode, Line charts present in the page are visible or not.

**Actual Result:** 
In High contrast Aquatic mode, Line charts present in the page are not visible in Windows.

**Refer Attachment:** 
1.In High contrast Aquatic mode, Line charts present in the page are not visible in Windows.png
2.In High contrast Aquatic mode, Line charts present in the page are not visible in Windows.mp4

https://github.com/user-attachments/assets/64e10e9c-49a1-42a2-869f-4fd30c1c5688

<img width="1937" alt="Image" src="https://github.com/user-attachments/assets/1012cb96-b82c-4c22-b5aa-5d401e0eb4d6" />

 
**Expected Result:**
In High contrast Aquatic mode, Line charts present in the page should be visible in Windows.

**Comment by KacperMadej** [comment]
Hi @msftedad 

Thank you for reporting the problem.

This is a website issue. The charts are working fine as far as I can see.

_Internal note:_
A dark background color makes the thumbnails not visible. With a slightly lighter color (customized default high contrast theme) the problem is partially visible:

![Image](https://github.com/user-attachments/assets/f0699392-bf13-4f3f-a2c2-bf7f17a26966)

**Comment by msftedad** [comment]
Hi @KacperMadej, deprecate old -ms-high-contrast and -ms-high-contrast-adjust properties in favor of Forced color properties. Move off these old high contrast properties to ensure ongoing high contrast support on Edge.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by KamilKubik** [comment]
Hello [msftedad](https://github.com/msftedad), we just resolved this issue! 

It was strictly related to our specific `mix-blend-mode` styles on the page, affecting the proper visibility. Could you also test and confirm it on your end?

**Comment by msftedad** [comment]
Hi @KamilKubik 
This issue is Fixed from our end. you can close this issue.
